src/main/java/uk/gov/ons/ctp/response/sample/service/SampleService.java

```diff
@@ -73,14 +79,18 @@ SampleSummary processSampleSummary(SurveyBase surveySampleObject, List<? extends
    */
   PartyDTO sendToPartyService(PartyCreationRequestDTO party) throws Exception;
 
+    Optional<SampleSummary> failSampleSummary(SampleSummary sampleSummary, String message);
```

Comment:
:art: Might just be github being weird but looks like this is indented

Reply:
LOL that's coz it's an intellij refactor - fixed now.

Reply:
The indentation is all over the place for all the java projects. We should just run a formatter over them all